modules/uncertainty/R/plots.R

```diff
@@ -50,20 +50,20 @@ plot_variance_decomposition <- function(plot.inputs,
   
   
   trait.plot <- base.plot + ggplot2::ggtitle("Parameter") + 
-    ggplot2::geom_text(ggplot2::aes(y = 1, x = points, label = trait.labels, hjust = 1), size = fontsize$axis/3) + 
+    ggplot2::geom_text(ggplot2::aes(y = 1, x = .data$points, label = trait.labels, hjust = 1), size = fontsize$axis/3) + 
     ggplot2::scale_y_continuous(breaks = c(0, 0), limits = c(0, 1)) + 
     ggplot2::theme(axis.text.x = ggplot2::element_blank())
   cv.plot <- base.plot + ggplot2::ggtitle("CV (%)") + 
-    ggplot2::geom_pointrange(ggplot2::aes(x = points, y = coef.vars, ymin = 0, ymax = coef.vars), size = 1.25) +
+    ggplot2::geom_pointrange(ggplot2::aes(x = .data$points, y = .data$coef.vars, ymin = 0, ymax = plot.data$coef.vars), size = 1.25) +
```

Comment:
One more `plot.data` -> `.data` needed -- not sure if I missed this before or if it somehow came back in the other fixes.

```suggestion
    ggplot2::geom_pointrange(ggplot2::aes(x = .data$points, y = .data$coef.vars, ymin = 0, ymax = .data$coef.vars), size = 1.25) +
```